### PR TITLE
chore: move get started into osx sidebar

### PR DESCRIPTION
**Pull request by banasa44**
## Description

Move the getStarted page to inside AragonOSx under Introduction to Aragon OSx in sidebar.
Done by adapting the react webpage to a .mdx compatible component

Task: [OS-1342](https://aragonassociation.atlassian.net/browse/OS-1342)

[OS-1342]: https://aragonassociation.atlassian.net/browse/OS-1342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/HeroHeader/index.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ interface IHeroHeader {
   img: ReactNode;
   imgStyle?: React.CSSProperties;
   imgAlt: string;
+  titleStyle?: React.CSSProperties; // Added titleStyle prop
+  subtitleStyle?: React.CSSProperties; // Added subtitleStyle prop
```
Comment (Rekard0) [inline src/components/HeroHeader/index.tsx:12]: are these comment are necessary?
```suggestion
  titleStyle?: React.CSSProperties;
  subtitleStyle?: React.CSSProperties;
```

File: versioned_docs/version-1.3.0/osx/01-get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: ''
+sidebar_label: Get Started
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+import DevTools from '../../../src/pages/dev-tools.tsx';
+
+<DevTools />
```
Comment (Rekard0) [inline versioned_docs/version-1.3.0/osx/01-get-started.mdx:10]: I only comment here
`devTools` naming seems a bit off, why we show devTools in get started.

i was expecting a structure like:
- getStarted
- - devToolds
- - ...etc

**Review by Rekard0 — COMMENTED** [review]
In general it is ok,
but regarding the styling ..etc perhaps using our own ODS would have been nicer

File: versioned_docs/version-1.3.0/osx/01-get-started.mdx (hunk 2 above)
Comment (banasa44) [reply versioned_docs/version-1.3.0/osx/01-get-started.mdx:10]: Dev-tools was the name (and web route path) of the get started webpage. https://devs.aragon.org/dev-tools/
I can rename everything to `getStarted`

**Review by Rekard0 — APPROVED** [review]

**Pull request merged**